pages/index.js

```diff
@@ -1,7 +1,35 @@
+import ArtPiecePreview from "@/components/ArtPiecePreview/ArtPiecePreview";
+import ArtPieces from "@/components/ArtPieces/ArtPieces";
+import useSWR from "swr";
+
+const URL = "https://example-apis.vercel.app/api/art";
+
+async function fetcher(url) {
+  const res = await fetch(url);
+
+  if (!res) {
+    const error = new Error(`An error occured...`);
+    error.info = await res.json();
+    error.status = res.status;
+    throw error;
+  }
+
+  return res.json();
```

Comment:
```suggestion
  const response = await fetch(url);

  if (!response) {
    const error = new Error(`An error occured...`);
    error.info = await response.json();
    error.status = response.status;
    throw error;
  }

  return response.json();
```

It would be better to avoid shorthands for better readability.